src/party_menu.c

```diff
@@ -923,76 +927,76 @@ static const u8 gUnknown_086159CE[] = {24, 25, 25, 25, 25, 25, 25, 25, 25, 26, 3
-static const u8 gUnknown_08615AB6[] = {11, 12};
-static const u8 gUnknown_08615AB8[] = {9, 10};
-static const u8 gUnknown_08615ABA[] = {4, 5, 6};
-static const u8 gUnknown_08615ABD[] = {1, 7, 8};
-static const u8 gUnknown_08615AC0[] = {1, 11, 12};
-static const u8 gUnknown_08615AC3[] = {59, 60};
-static const u8 gUnknown_08615AC5[] = {75, 76};
-static const u8 gUnknown_08615AC7[] = {57, 58};
-static const u8 gUnknown_08615AC9[] = {73, 74};
-static const u8 gUnknown_08615ACB[] = {89, 90};
-static const u8 gUnknown_08615ACD[] = {52, 53, 54};
-static const u8 gUnknown_08615AD0[] = {68, 69, 70};
-static const u8 gUnknown_08615AD3[] = {84, 85, 86};
-static const u8 gUnknown_08615AD6[] = {116, 117, 118};
-static const u8 gUnknown_08615AD9[] = {132, 133, 134};
-static const u8 gUnknown_08615ADC[] = {148, 149, 150};
-static const u8 gUnknown_08615ADF[] = {100, 101, 102};
-static const u8 gUnknown_08615AE2[] = {49, 55, 56};
-static const u8 gUnknown_08615AE5[] = {65, 71, 72};
-static const u8 gUnknown_08615AE8[] = {81, 87, 88};
-static const u8 gUnknown_08615AEB[] = {97, 103, 104};
-static const u8 gUnknown_08615AEE[] = {161, 167, 168};
-static const u8 gUnknown_08615AF1[] = {17, 27, 28};
-
-static const u8 *const gUnknown_08615AF4[] =
-{
-    gUnknown_085E9E43,
-    gUnknown_085EA010,
-    gUnknown_085EA02A,
-    gUnknown_085E9E55,
-    gUnknown_085E9E64,
-    gUnknown_085E9E79,
-    gUnknown_085E9E8F,
-    gUnknown_085E9EBC,
-    gUnknown_085E9ED4,
-    gUnknown_085E9EE9,
-    gUnknown_085E9FDB,
-    gUnknown_085EA046,
-    gUnknown_085EA05B,
-    gUnknown_085E9F01,
-    gUnknown_085E9F58,
-    gUnknown_085E9F6F,
-    gUnknown_085E9F81,
-    gUnknown_085E9F90,
-    gUnknown_085E9FA7,
-    gUnknown_085E9FC2,
+static const u8 sPaletteBufferOffsets_GenderIcon[] = {11, 12};
+static const u8 sPaletteBufferOffsets_HPBar[] = {9, 10};
+static const u8 sPaletteBufferOffsets_PartySlotBG[] = {4, 5, 6};
+static const u8 sPaletteBufferOffsets_PartySlotBorder[] = {1, 7, 8};
+static const u8 sPaletteBufferOffsets_EmptyPartySlot[] = {1, 11, 12}; //includes border and background
+static const u8 sPaletteBufferIds_MaleGenderIcon[] = {59, 60};
+static const u8 sPaletteBufferIds_FemaleGenderIcon[] = {75, 76};
+static const u8 sPaletteBufferIds_GreenHPBar[] = {57, 58};
+static const u8 sPaletteBufferIds_YellowHPBar[] = {73, 74};
+static const u8 sPaletteBufferIds_RedHPBar[] = {89, 90};
+static const u8 sPaletteBufferIds_NormalPartySlotBG[] = {52, 53, 54};
+static const u8 sPaletteBufferIds_DisqualifiedPartySlotBG[] = {68, 69, 70};
+static const u8 sPaletteBufferIds_FaintedPartySlotBG[] = {84, 85, 86};
+static const u8 sPaletteBufferIds_SelectedPartySlotBG[] = {116, 117, 118};
+static const u8 sPaletteBufferIds_DisqualifiedSelectedPartySlotBG[] = {132, 133, 134};
+static const u8 sPaletteBufferIds_FaintedSelectedPartySlotBG[] = {148, 149, 150};
+static const u8 sPaletteBufferIds_EnteredPartySlotBG[] = {100, 101, 102}; //entered? As in into a contest or battle?
```

Comment:
Remove comment; name should be descriptive enough.

Reply:
But it clearly isn't, if it's leading to confusion...